tests/unit_tests/test_invalid_build.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+
+from magniv.build.build import build
+from tests.unit_tests.fixtures.test_files import (
+    INVALID_KWARG_FILE,
+    INVALID_TASK_KEY_FILE,
+    INVALID_TRIGGER_FILE,
+)
+
+# TODO: merge with TestBuild upon refactor there
+
+
+class TestBuildInvalid:
+    @pytest.fixture
+    def folder(self, tmpdir, file):
+        tmpdir.mkdir("tasks").join("main.py").write(file)
+        tmpdir.join("tasks/requirements.txt").write("magniv")
+        return str(tmpdir.join("tasks"))
+
+
+class TestBuildInvalidKwarg(TestBuildInvalid):
+    @pytest.fixture(scope="class")
```

Comment:
Double check if we need the scope, if not remove?

Reply:
we don't; good call, removed